nss/src/group/mod.rs

```diff
@@ -27,7 +27,8 @@ impl GroupHooks for AuthdGroup {
 
 /// get_all_entries connects to the grpc server and asks for all group entries.
 fn get_all_entries() -> Response<Vec<Group>> {
-    super::RT.block_on(async {
+    let rt = tokio::runtime::Builder::new_current_thread().enable_all().build().unwrap();
+    rt.block_on(async {
```

Comment:
An idiomatic nitpick: In Rust, the types are usually imported at the beginning of the file, so instead of using the fully qualified path `tokio::runtime::Builder`, it's more idiomatic to:
```Rust
// Import section
use tokio::runtime::Builder;

// Inside {all} the changed functions
let rt = Builder::new_current_thread().enable_all().build().unwrap(); //...
```